devops/nginx/default.conf

```diff
@@ -0,0 +1,48 @@
+server {
+  listen 80;
+  location / {
+    return 301 https://$host$request_uri;
+  }
+
+}
+server {
+  listen 443 ssl;
+  ssl on;
+  ssl_certificate /etc/nginx/conf.d/origin.crt;
+  ssl_certificate_key /etc/nginx/conf.d/origin.key;
+  server_name dev.ev-dashboard-cpo.energyweb.org;
```

Comment:
Please add comment, that it's static here because for prod and other envs we will move app to k8s

Reply:
Added